hexa/pipeline_templates/schema/mutations.py

```diff
@@ -74,4 +75,40 @@ def resolve_create_pipeline_template_version(_, info, **kwargs):
     return {"pipeline_template": pipeline_template, "success": True, "errors": []}
 
 
+@pipeline_template_mutations.field("createPipelineFromTemplateVersion")
+def resolve_create_pipeline_from_template_version(_, info, **kwargs):
+    request: HttpRequest = info.context["request"]
+    input = kwargs["input"]
+
+    workspace = get_workspace(request.user, input.get("workspace_slug"))
+    if not workspace:
+        return {"success": False, "errors": ["WORKSPACE_NOT_FOUND"]}
+
+    if not request.user.has_perm("pipelines.create_pipeline", workspace):
+        return {"success": False, "errors": ["PERMISSION_DENIED"]}
+
+    try:
+        template_version = PipelineTemplateVersion.objects.get(
+            id=input["pipeline_template_version_id"]
+        )
+    except PipelineTemplateVersion.DoesNotExist:
+        return {"success": False, "errors": ["PIPELINE_TEMPLATE_VERSION_NOT_FOUND"]}
+
+    pipeline_code = f"{template_version.template.source_pipeline.code} (from Template)"
```

Comment:
Can you give more context about this comment? The (from Template) would be appended to the pipeline code and would break the parsing if we would reuse it as it.
So I would like to know if the expected flow is for the user to open the pipeline code and edit it, and the comment line will indicate that code is copied form template and has to be edited? 
If the goal of it is to mark it as code form tempalte then i would append it as a python comment "/n # (from Template)"

Reply:
Thanks for raising this

The intend was to create a different name for the pipeline that is created but you are correct that it makes the code unparseable. In addition to mis-align the python code and the Database object (the `@pipeline(code= , name= )` would need to be modified )  

So, as part of https://github.com/BLSQ/openhexa-app/pull/885, I am using the same name as the original Pipeline to avoid such complexity at this point